common.yml

```diff
@@ -2203,4 +2322,11 @@ buildvariants:
     resmoke_use_tls: _tls
     resmoke_use_ssl: _ssl
     USE_SSL: "true"
-  tasks: *ubuntu_x86_64_race_tasks
+  tasks: 
+  - name: "unit"
+  - name: ".3.4"
+  - name: ".3.6"
+  - name: ".4.0"
+  - name: ".4.2"
+  - name: ".4.4"
+  - name: "dist"
```

Comment:
since we don't release on the race-detector buildvariant, is there any particular reason we need to run the `dist` task?